lib/api/apiUtils/object/storeObject.js

```diff
@@ -0,0 +1,81 @@
+import { errors } from 'arsenal';
```

Comment:
this is all just moved from services with the functions there remaining?  will you please add a note in services.js that these are to be deleted once references are updated in the other api's and open an issue for that to be done?